core/base/dimensionReduction/DimensionReduction.h

```diff
@@ -63,6 +70,8 @@ namespace ttk {
       ISOMAP = 4,
       /** Principal Component Analysis */
       PCA = 5,
+      /** Topological Mapper */
```

Comment:
I'd replace "Topological Mapper" by "TopoMap" (otherwise, it could be confused with the mapper, which is something else).